ultravox/tools/ds_tool.py

```diff
@@ -78,12 +96,18 @@ def _map_sample(self, sample):
 # Ex: just ds_tool tts -d google/boolq -u fixie-ai/boolq-audio -c question -a audio --token $HF_WRITE_TOKEN
 # Ex: just ds_tool textgen -d fixie-ai/boolq-audio -u fixie-ai/boolq-audio -c explanation
 # Ex: just ds_tool textgen -d ylacombe/expresso -u fixie-ai/expresso -c continuation -T @expresso_template.txt
+# Ex: just ds_tool textgen -d allenai/soda --shuffle True --split train -n 10000 -u fixie-ai/soda -c alt_last_turn -j -m llama3-8b -T "dialogue[:-1]"
+# Ex: just ds_tool textgen -d allenai/soda --shuffle True --split validation -n 1000 -u fixie-ai/soda -c alt_last_turn -j -m llama3-8b -T "dialogue[:-1]"
+# Ex: just ds_tool textgen -d allenai/soda --shuffle True --split test -n 1000 -u fixie-ai/soda -c alt_last_turn -j -m llama3-8b -T "dialogue[:-1]"
```

Comment:
I think Jinja can do both the expression handling and formatting, e.g., `{{dialogue[:-1]}}`. Not having to have two separate modes would be nice...

Reply:
With Jinja the result is going to be a string, not a list, right?
I don't see how I can cleanly then re-create the list of turns to give to the ChatCompletion API.

Reply:
It's a string, of course, but it can be in JSON format and probably auto-converted to a sequence accordingly.
e.g.
```
template = Template('''
[
{% for person in data %}
  {
    "name": "{{ person.name }}",
    "age": {{ person.age }}
  }{% if not loop.last %},{% endif %}
{% endfor %}
]
''')
```

Reply:
Using Jinja now, but I had to use a slightly weird try/catch method for it. PTAL.